server/player/player.go

```diff
@@ -840,6 +841,27 @@ func (p *Player) SetMobile() {
 	p.updateState()
 }
 
+// CanClimb checks if the player can climb ladders & vines.
+func (p *Player) CanClimb() bool {
+	return p.canClimb.Load()
+}
+
+// SetCanClimb allows a player to climb ladders & vines.
+func (p *Player) SetCanClimb() {
+	if !p.canClimb.CAS(false, true) {
+		return
+	}
+	p.updateState()
+}
+
+// SetCantClimb disallows a player from climbing ladders & vines.
+func (p *Player) SetCantClimb() {
```

Comment:
These have rather awkward names. Maybe `EnableClimbing` and `DisableClimbing` would be better? I'm actually not sure if exposing these methods has any real use at all. Is there a good example where you'd want to disable climbing for players?